### PR TITLE
Create walnut creek-ca.md

### LINKED ISSUE #2: AMZod favorite places to eat
Let's add a document using:
- [x] Create a branch
- [x] Add a new file
- [x] Commit my file
- [x] Open Pull Request
- [ ] Discuss
- [ ] Merge

### PR DESCRIPTION
**Pull request by AMZod**
Closes #2 by adding a file for places to eat in Walnut Creek, CA.

### CODE REVIEW

File: eats/walnut creek-ca.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Walnut Creek, CA Great Places to Eat
+
+* Monecatini's
+* Va di Vi Bistro
```
Comment (AMZod) [inline eats/walnut creek-ca.md:4]: Delicious French Bistro with small plates

File: eats/walnut creek-ca.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+* 1515
```
Comment (AMZod) [inline eats/walnut creek-ca.md:5]: Kind of a Lounge/Restaurant

File: eats/walnut creek-ca.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+### Walnut Creek, CA Great Places to Eat
+
+* Monecatini's
```
Comment (AMZod) [inline eats/walnut creek-ca.md:3]: Italian!

**Comment by rod11** [comment]
Hello

**Comment by carolynshin** [comment]
:sheep: it!

**Comment by eichnera** [comment]
:sheep: it

**Pull request merged**